tests/integration/Filters.test.js

```diff
@@ -0,0 +1,331 @@
+        // Default values are 15, 200
+        filters.vm.$data.range = [60,200];
+
+        let filterDishesBtn = wrapper.find("#filter-dishes-btn");
+        await filterDishesBtn.trigger("click");
+        
+        let filteredDishes = wrapper.findAll(".filtered-items .dish-inner").length;
+
+        expect(filteredDishes).toBe(3);
+    });
+
+    it("max price slider changes value on input", async () => {
+
+        wrapper = await mountMenu(dishes);
+
+        // Open dialog box first
+        let filterButton = wrapper.find("button.mx-2");
+        await filterButton.trigger("click");
+        
+        // range slider is not compatible with tests; it has to be bypassed
+        let filters = wrapper.findComponent(Filters);
+
+        // Default values are 15, 200
+        filters.vm.$data.range = [15,70];
+
+        let filterDishesBtn = wrapper.find("#filter-dishes-btn");
+        await filterDishesBtn.trigger("click");
+        
+        let filteredDishes = wrapper.findAll(".filtered-items .dish-inner").length;
+
+        expect(filteredDishes).toBe(4);
+    });
+
+
+    it("closes dialog window on filter", async () => {
+
+        wrapper = await mountMenu(dishes);
+
+        let filterButton = wrapper.find("button.mx-2");
+
+        expect(filterButton.exists()).toBeTruthy();
+        expect(filterButton.find("i.mdi-filter")).toBeTruthy();
+
+        // Open Dialog
+        await filterButton.trigger("click");
+
+        // Click Filter Dishes Btn
+        let filterDishesBtn = wrapper.find("#filter-dishes-btn");
+        await filterDishesBtn.trigger("click");
+
+        let dialog = wrapper.find(".v-dialog");
+
+        expect(dialog.classes('v-dialog--active')).toBe(false);
+    });
+
+    it("closes dialog window on close window", async () => {
+
+        wrapper = await mountMenu(dishes);
+
+        let filterButton = wrapper.find("button.mx-2");
+
+        expect(filterButton.exists()).toBeTruthy();
+        expect(filterButton.find("i.mdi-filter")).toBeTruthy();
+
+        // Open Dialog
+        await filterButton.trigger("click");
+
+        // Click Filter Dishes Btn
+        let closeFiltersBtn = wrapper.find("#dialog-close-btn");
+        await closeFiltersBtn.trigger("click");
+
+        let dialog = wrapper.find(".v-dialog");
+
+        expect(dialog.classes('v-dialog--active')).toBe(false);
+    });
+
+    it("it clears filters and shows all default items", async () => {
+
+        wrapper = await mountMenu(dishes);
+
+        // Open dialog box first
+        let filterButton = wrapper.find("button.mx-2");
+        await filterButton.trigger("click");
+        
+        // range slider is not compatible with tests; it has to be bypassed
+        let filters = wrapper.findComponent(Filters);
+
+        // Default values are 15, 200
+        filters.vm.$data.range = [15,70];
+
+        // Filter Dishes
+        let filterDishesBtn = wrapper.find("#filter-dishes-btn");
+        await filterDishesBtn.trigger("click");
+
```

Comment:
Ensure that the filters were applied and made a different in conditioning the result (expect all-items to not exist)

Reply:
I need elaboration on this. I do not get what you mean.